terraform/environment/cloudwatch_alarms.tf

```diff
@@ -54,7 +54,7 @@ resource "aws_cloudwatch_log_metric_filter" "csrf_mistmatch_filter" {
     namespace = "online-lpa/Cloudwatch"
     value     = "1"
   }
-}
+} */
```

Comment:
is this commenting out intentional?

Reply:
yes - see thread in slack. we have issues in the dev environment, that needed urgent attention.